### ISSUE TITLE
Different table row hover background colors in dark/light mode

### ISSUE THREAD

**Issue by Varr1ck** [jothepro/doxygen-awesome-css#60]
Is this even possible currently?
There is no css custom property to overwrite.

**Comment by jothepro** [comment]
Can you give me more information on what you need? I'm not sure I understand what you need. I'm not aware of any hover effect on tables in Doxygen. Do you want to add such a hover effect or do you just want to adjust existing behavior that I am not aware of right now?

**Comment by Varr1ck** [comment]
I was looking for that table hover highlight effect which one can do with CSS.
I can add such behaviour for all tr elements in the custom.css, but I don't think that hover highlight color can be chosen for dark/light theme individually in an easy way, am I right?

**Comment by jothepro** [comment]
That's right, there currently is no CSS-variable to configure table row highlighting. If you think that this adds value to the theme and you found a good solution to implement it, you are welcome to open a PR! 😊
